### PR TITLE
UI_20150317_000001 to UI_20150317_000052

### PR DESCRIPTION
**Pull request by Adamai**
UI_20150317_000020	{@st59}Fechar informação da Quest{/}
Quest should be kept in english as it is known as a common gamming expression.

UI_20150317_000021	Availest
UI_20150317_000027	{@st41b}Bidding Price
Sorry, can't remember how to translate those exactly

### CODE REVIEW

File: UI_BP.tsv [hunk 1]
```diff
@@ -1,55 +1,55 @@
...
+UI_20150317_000018	{memo X}{@st41b}Itens vendidos : ×%s, Itens Mantidos : ×%s{/}
+UI_20150317_000019	{@st43}{s22}Leilão de Itens{/}
+UI_20150317_000020	{@st59}Fechar informação da Quest{/}
+UI_20150317_000021	Availest
+UI_20150317_000022	{@st41b}Tempo Restante
+UI_20150317_000023	3 horas 16 minutos
+UI_20150317_000024	{@st41b}Preço Inicial
+UI_20150317_000025	{@st41b}Ofertas
+UI_20150317_000026	{@st41b}Preço Atual
+UI_20150317_000027	{@st41b}Bidding Price
```
Comment (wevtimoteo) [inline UI_BP.tsv]: WDYT `Lance`?

**Comment by Adamai** [comment]
Deus do céu como n percebi isso antes, BOA!

**Comment by wevtimoteo** [comment]
:rocket:

**Comment by imcgames** [comment]
Thank you :+1:

**Comment by Adamai** [comment]
It is an honor to help :)

Check this pal's UI fully translated(he's doing an amazing work), I already sent him some suggestions and he keeps updating it with better expressions and corrections. 
https://github.com/Treeofsavior/PortugueseTranslation/pull/16

**Pull request merged**